editor/editor_help.cpp

```diff
@@ -1930,17 +1951,15 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 
 		} else if (doc->class_list.has(tag)) {
 			// Class reference tag such as [Node2D] or [SceneTree].
-			// Use monospace font with translucent colored background color to make clickable references
+			// Use monospace font to make clickable references
```

Comment:
Punctuation should be added here.

I read through the code, looks good to me.

Reply:
Me and my homies love proper punctuation on old comments changed on the way.

Reply:
I looked at this again, I'm not sure if punctuation is necessary? It is done to make the comment more readable, but feel free to contest that.